src/Classes/Command/SatisCommand.php

```diff
@@ -70,8 +65,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     'type' => 'composer',
                     'url' => 'file://' . $webroot . '/Web/packages-TYPO3Extensions-quarter.json'
                 )
-            ),
-            $this->fetchRepositories()
+            )
```

Comment:
Looks like you can remove the `array_merge` call above as well and just define the array directly?